### ISSUE TITLE
Synced mail client

### ISSUE THREAD

**Issue by KaKi87** [open-source-ideas/open-source-ideas#243]
## Project description

There are many mail clients available for every platform, but none that supports all platforms and sync between these.
For instance, I'd like to be able to login an e-mail account on my PC and automatically see it appear on my smartphone.
The only credentials I'll have to type are username and password for the app, once per device, while the rest is synced.

## Relevant Technology

Platforms : Windows, Linux, Mac, Android, iOS

It could also be web-only though, but native would be preferred.

### Complexity

- [ ] Beginner - This project requires no or little prior knowledge of the technolog(y|ies) specified to contribute to the project
- [x] Intermediate - The user should have some prior knowledge of the technolog(y|ies) to the point where they know how to use it, but not necessarily all the nooks and crannies of the technology
- [ ] Advanced - The project requires the user to have a good understanding of all components of the project to contribute

### Required time (ETA)

- [ ] Little work - A couple of days
- [x] Medium work - A week or two
- [ ] Much work - The project will take more than a couple of weeks and serious planning is required

### Categories

- [x] Mobile app
- [ ] IoT
- [x] Web app
- [x] Frontend/UI
- [ ] AI/ML
- [x] APIs/Backend
- [ ] Voice Assistant
- [ ] Developer Tooling
- [ ] Extension/Plugin/Add-On
- [ ] Design/UX
- [ ] AR/VR
- [ ] Bots
- [ ] Security
- [ ] Blockchain
- [ ] Futuristic Tech/Something Unique

**Comment by FredrikAugust** [comment]
> For instance, I'd like to be able to login an e-mail account on my PC and automatically see it appear on my smartphone.

_What_, would you like to see appear automatically?

**Comment by KaKi87** [comment]
> _What_, would you like to see appear automatically?

The newly added e-mail address.

Example use case :
1. I'm on my PC and I create an account on the e-mail client. I add my personal e-mail address to it.
2. I log in the e-mail client from my smartphone. Thanks to the synced e-mail client, I can now access my personal e-mail address from it.
3. I'm at my work office and I login on the e-mail client from my workstation. I add my professional e-mail address to it.
4. Suddenly I have to stop working at the office and I must work from home. Thanks to the synced e-mail client, I already can access my professional e-mail address from home on my PC and on my smartphone without re-authenticating again on both.

**Comment by GrpeApple** [comment]
Why not add a biometric feature to avoid someone getting in?

**Comment by KaKi87** [comment]
This project would work like a password manager, meaning the synced accounts credentials would be encrypted with a master password and that one itself won't be resettable.
The user really has nothing to loose if he looses the master password, except the time he'll spend to re-creating a new account.

**Comment by GrpeApple** [comment]
Then my point still stands, use a **biometric** _feature_ if the **master** **password** is a _hassle_ to write, like me. Who has _115_ characters in a **master** **password.**
So I often store the **master** **password** to another one,
the password manager has a **master** **password** with random characters or generated at a length of 40 on the _original_ **password** _manager_ which doesn’t make any sense.
So I want a **biometric** _feature_ on the **password** _manager._

**Comment by virtadpt** [comment]
@GrpeApple Your password strategy kind of doesn't make sense in light of modern password guidelines.  If you can't remember your master password because its structure isn't cognitive (i.e., it's not something that you can remember, like a sentence)... what the heck is your threat model?

**Comment by GrpeApple** [comment]
> @GrpeApple Your password strategy kind of doesn't make sense in light of modern password guidelines.  If you can't remember your master password because its structure isn't cognitive (i.e., it's not something that you can remember, like a sentence)... what the heck is your threat model?

# TL;DR, Too Long; Didn't Read: I do not make any sense.

Well, then. I will upgrade my password generator to 115 characters. On that different password manager.
I can remember my master password, and if I cannot remember, I have to use many backup plans that does not make sense anymore.
It does not make sense that I will have to remember my 115 character password, but I enjoy doing that and it has benefits too.
What I am saying is that I enjoy it.

**Comment by GrpeApple** [comment]
> @GrpeApple Your password strategy kind of doesn't make sense in light of modern password guidelines. If you can't remember your master password because its structure isn't cognitive (i.e., it's not something that you can remember, like a sentence)... what the heck is your threat model?

Oh.... Thank you.
Indeed my password strategy doesn’t quite make sense.
But for users who want to do what they like, a biometric feature can help. In my opinion, of course.

**Comment by AppleGrpe** [comment]
Sounds interesting......
